app/components/supplementary-file-browser.js

```diff
@@ -0,0 +1,62 @@
+import Ember from 'ember';
+import $ from 'jquery';
```

Comment:
You can also access `Ember.$` (global jquery) or `this.$.()` (selector for elements inside this container)